compiler/rustc_mir_build/src/build/scope.rs

```diff
@@ -618,6 +618,16 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
             }
         } else {
             assert!(value.is_none(), "`return` and `break` should have a destination");
+            // `continue` statements generate no MIR statement with the `continue` statement's Span,
+            // and the `InstrumentCoverage` statement will have no way to generate a coverage
```

Comment:
I could add a check if instrument coverage is enabled and only add the dummy assignment then

Reply:
done